tests/Controllers/BeatmapDiscussionsControllerTest.php

```diff
@@ -74,12 +77,12 @@ public function testPutVoteChange()
         $this
             ->actingAs($this->anotherUser)
             ->put(route('beatmap-discussions.vote', $this->beatmapDiscussion), [
-                'beatmap_discussion_vote' => ['score' => '-1'],
+                'beatmap_discussion_vote' => ['score' => '1'],
```

Comment:
name and description need updating in this case...?

and maybe this test done by bng.